README.md

```diff
@@ -6,8 +6,8 @@ system can be used to perform the benchmarking, e.g.
 
 # Installation
 
-First run the `build.sh` script which should perform most, if not all, of the
-installation work needed.
+Running `./build.sh` should perform most, if not all, of the installation work
```

Comment:
I know this isn't related, but I wonder why we say "should" here? Does the script do the install or not? :P

Reply:
I didn't write that part of the sentence. I assumed it was an Edd special :)

I guess the reason for the weasel wording is that we never quite know if we've checked for all the dependencies. But I agree, I'm not sure such equivocation serves a useful purpose.

Reply:
Hey! I didn't write any of the docs for warmup_stats :P That line is your change from `b43469f17`, "Minimally update the README".

Shall we reword it while we are here?

Reply:
I stand corrected. I *did* write that. Mea culpa!

In fairness, I condensed a long paragraph which said the same thing down into that one. I've now tried simplifying it a bit. See what you think.